README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The tests suite:
+- Must be easy to configure through settings files to define the substra network to target
+- Must be independent from the substra network
+  - Substra network must be started separately
+  - Substra network can be located locally / on the cloud
+  - Substra network can be started through docker-compose or through kubernetes
+- Requires a setup with at least 2 organisations
+- Should be able to run multiple times without restarting the network
+
+Each new test:
+- Should have a deterministic behavior (not fail randomly)
+- Must be fast to run:
+  - avoid adding latency
```

Comment:
"Latency" seems a bit confusing here, since it could refer to network latency. What about "avoid tests that take a lot of time to complete" ?

Reply:
What about capitalizing these sub-bullet points, to make it consistent with the rest?

Reply:
> A palanquée of extra comments. Looks good!
> 
> Feel free to pick and choose among my suggestions.
> 
> Also, I would recommend ordering the items by descending order of importance, if that's not already the case.

Thanks a lot for your comments, it's better and better!
And it's important to have clear guidelines :)